libfreerdp/core/capabilities.c

```diff
@@ -1351,6 +1351,10 @@ static BOOL rdp_apply_input_capability_set(rdpSettings* settings, const rdpSetti
 		}
 		if (settings->HasExtendedMouseEvent)
 			settings->HasExtendedMouseEvent = src->HasExtendedMouseEvent;
+		if (settings->HasRelativeMouseEvent)
+			settings->HasRelativeMouseEvent = src->HasRelativeMouseEvent;
```

Comment:
Uhm, this condition + condition handling was duplicated from above

Reply:
no, this is the client - server ping pong.
only if locally enabled even consider accepting the value of the remote

Reply:
Oh, right indeed. Both setting names have the same length and I did not realize, that the first one was about the extended mouse event and the second about the relative one (for some reason it looked to me as line duplication).

> only if locally enabled even consider accepting the value of the remote

yeah, right.